src/network/connection_dispatcher_task.cpp

```diff
@@ -19,12 +19,12 @@ ConnectionDispatcherTask::ConnectionDispatcherTask(
       thread_registry_(thread_registry),
       interpreter_provider_(interpreter_provider),
       next_handler_(0) {
-  RegisterEvent(listen_fd, EV_READ | EV_PERSIST,
-                METHOD_AS_CALLBACK(ConnectionDispatcherTask, DispatchPostgresConnection), this);
+  RegisterEvent(listen_fd, EV_READ | EV_PERSIST, METHOD_AS_CALLBACK(ConnectionDispatcherTask, DispatchConnection),
+                this);
   RegisterSignalEvent(SIGHUP, METHOD_AS_CALLBACK(NotifiableTask, ExitLoop), this);
 }
 
-void ConnectionDispatcherTask::DispatchPostgresConnection(int fd, int16_t) {  // NOLINT
+void ConnectionDispatcherTask::DispatchConnection(int fd, int16_t) {  // NOLINT
```

Comment:
Add a TODO that `DispatchConnection` will also need to take an argument telling it which interpreter to use for the connection.

Reply:
Updated